### PR TITLE
Revert "feat: Set org-id and project-id as tags for outcome"

### PR DESCRIPTION
**Pull request by untitaker**
Reverts getsentry/sentry#20410

### CODE REVIEW

**Comment by untitaker** [comment]
@marbon87 we have some concerns about the load/bills this creates on our statsd consumer. We'll revert for now. Not sure about your usecase but it might be cleaner to write a kafka consumer and read from the outcomes topic to get all the data.

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -1120,11 +1120,7 @@ def discard_event(job, attachments):
     metrics.incr(
         "events.discarded",
         skip_internal=True,
-        tags={
-            "organization_id": project.organization_id,
-            "platform": job["platform"],
-            "project_id": job["project_id"],
-        },
+        tags={"organization_id": project.organization_id, "platform": job["platform"]},
```
Comment (tonyo) [inline src/sentry/event_manager.py:1123]:
```suggestion
        tags={"platform": job["platform"]},
```

Let's also remove this while we're on it.

**Review by tonyo — APPROVED** [review]

**Comment by marbon87** [comment]
Setting some tags should be too expensive shouldn't it?
My usecase is that I want to have a better overview which project creates the most load on the system by using the metrics.

**Comment by untitaker** [comment]
>Setting some tags should be too expensive shouldn't it?

in our case it is, and we've gone to great lenghts to get rid of project and org tags from metrics already... if you want a way forward I'd recommend adding a django setting in `conf/server.py` to control this behavior.

**Comment by marbon87** [comment]
Like this? https://github.com/getsentry/sentry/pull/20477

**Pull request merged**